packages/airnode-deployer/.gitignore

```diff
@@ -8,9 +8,9 @@
 coverage/
 
 # Config files
```

Comment:
It would be nice to mention that this is deliberate so the next me won't update this.